dali/python/nvidia/dali/ops.py

```diff
@@ -394,7 +410,8 @@ def __init__(self, inputs, op, **kwargs):
                 self._inputs = list(self._inputs) + [arg_inp]
 
         if self._op.schema.IsDeprecated():
-            use_instead = self._op.schema.DeprecatedInFavorOf()
+            # TODO(klecki): how to know if this is fn or ops?
+            use_instead = _op_name(self._op.schema.DeprecatedInFavorOf(), "fn")
```

Comment:
I think we could pass this information to the op class constructor when we register it.